Examples/Example.Makefile

```diff
@@ -8,15 +8,19 @@ BAZEL_WRAPPER=$(RULES_PODS_DIR)/tools/bazelwrapper
 # Currently `bazelwrapper` relies on pwd, which causes issues here
 BAZEL=~/.bazelenv/versions/0.18.0/bin/bazel
 
+BAZEL_OPTS=--override_repository=rules_pods=$(RULES_PODS_DIR) \
+		--spawn_strategy=standalone \
```

Comment:
Part of the point of this CI is to validate hermaticness of these builds ( so it shouldn't run as `standalone` )